torch/nn/quantized/reference/modules/conv.py

```diff
@@ -0,0 +1,19 @@
+# flake8: noqa: F401
```

Comment:
Should we keep these files under `torch/nn/quantized/_reference`? I guess there aren't existing users of this namespace?

Reply:
Never mind, just saw your comment above